src/main/resources/mappings/master-key.json

```diff
@@ -0,0 +1,11 @@
+{
+    "dynamic": false,
+    "_meta": {
+      "schema_version": 1
+    },
+    "properties": {
+      "master_key": {
```

Comment:
nit: maybe we should change to config index to allow for more flexible adjustments in the future.

Reply:
I have changed this to `.plugins-flow-framework-config`